lib/vagrant.rb

```diff
@@ -52,6 +52,13 @@
     end
     Log4r::Outputter.stderr.formatter = Vagrant::Util::LoggingFormatter.new(base_formatter)
     logger = nil
+
+    # Cloud gem uses RestClient to make HTTP requests, so
+    # log them if debug is enabled
+    if level == 1
+      # TODO: Need to ensure token is marked sensitive, if possible here
+      ENV["RESTCLIENT_LOG"] = "stdout"
```

Comment:
So, I could remove this. It's tricky because the restclient gem will dump the http request. Which in this case, will include folks Vagrant Cloud tokens. I don't know if our `vagrant.sensitive` setting would prevent this.

Reply:
But I do think it would be useful to know what HTTP requests are happening in the debug log. If we can some how mute/remove tokens from the URL that would be great.

Reply:
So we should update this to be a Log4r instance since we'll have scrubbing in the logs as well.